witness/src/Dscp/Witness/Web/Wallet/API.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE DataKinds     #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Dscp.Witness.Web.Wallet.API
+     ( WalletEndpoints (..)
+     , WalletAPI
+     , walletAPI
+     ) where
+
+import Servant
+import Servant.Generic
+
+import Dscp.Core.Types
+import Dscp.Witness.Web.Wallet.Types
+
+-- | API/implementation of wallet endpoints.
+data WalletEndpoints route = WalletEndpoints
+    { -- shortened prefix for purpose, one will most probably use type aliases
+      -- refering this type
+      wGetNextNonce :: route
```

Comment:
Why not return the nonce with the account state?

Reply:
Oki, fixed